tests/pipelines/test_file_processing.py

```diff
@@ -30,6 +31,14 @@ def test_get_file_paths_from_dir_recursive(testdir_file_path):
     for fpath in fileproc.file_paths:
         assert fpath.endswith(".json")
 
+def test_move_all_filepaths():
+    fileproc = FileProcessing()
+    with tempfile.TemporaryDirectory() as temp_dir_one:
+        temp_dir_two = tempfile.TemporaryDirectory()
+        temp = tempfile.NamedTemporaryFile(dir=temp_dir_one, suffix='.pdf')
+        print(temp.name)
+        fileproc.move_all_file_paths(temp_dir_one,temp_dir_two.name)    
```

Comment:
why do you use `temp_dir_one` vs `temp_dir_two.name`?

Reply:
weird quirk of tempdir, because I start this using "with tempdir as tempdir_one" it uses its name automatically, 
but when I create tempdir_two it needs to be told to look for its name attribute.